linux/net/rina/ipcps/Makefile

```diff
@@ -10,5 +10,7 @@ ccflags-y += -Wtype-limits
 
 obj-$(CONFIG_RINA_NORMAL_IPCP)    += normal-ipcp.o
 obj-$(CONFIG_RINA_SHIM_ETH_VLAN)  += shim-eth-vlan.o
+shim-eth-vlan-y :=				\
```

Comment:
This should probably be

shim-eth-vlan-$(CONFIG_RINA_SHIM_ETH_VLAN) :=

otherwise I think you get an error when not compiling in shim-eth-vlan